ios/RNModules/RNLogging/RNLogging/RNLoggingModule.m

```diff
@@ -45,29 +31,61 @@ + (BOOL)requiresMainQueueSetup {
     return YES;
 }
 
-- (id<RNLogger>)logger {
-    if (!_logger) {
-        return [[RNDummyLogger alloc] init];
```

Comment:
This is useful in dev environment.

Reply:
Returned to the previous state. I tested it and it worked but please let me know in case of any problem... :)